.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "python.pythonPath": "~/locus_dev/devel/share/aiorospy/venv/bin/python",
```

Comment:
100% in favour of checking these in. Especially as we're generally all vscode users and this saves us all from having to configure.

It does feel like we're leaking our workspace details into the public realm. At the very least a little irritating for other contributors.  I don't think anything needs to be done about this at this point.

Reply:
yeah, https://github.com/Microsoft/vscode/issues/57551 looks like the problem